### PR TITLE
Accessibility of Portfolio Project

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
**In this milestone,**

Created a new branch and collaborated on this milestone
Made Accessibility changes to the Portfolio Project
Tested accessibility instructions one by one

Below are the results of the website's evaluation:
- Title name - Document
Link: https://github.com/Bellagirl-maker/Portfolio/pull/8/files#r1128000562

- Image text alternatives - one alert was found and the problem was the same alt name was used
Link: https://github.com/Bellagirl-maker/Portfolio/pull/8/files#r1127719622

- Text headings - one warning was found about lacking a h1 heading tag. 
Link: https://github.com/Bellagirl-maker/Portfolio/pull/8/files#r1127716374

- Color contrast - found problems with color contrast in p tag, h4 tag, button tag and textarea tag. Contrast ratios were lower than required ratios. Required ratio for all tags was 4.5:1. Contrast ratio was increased for all tags to pass the requirement.  
Link: https://github.com/Bellagirl-maker/Portfolio/pull/8/files#r1127728085

- Resize - no issues found
- Interaction – no issues found
- Moving content – no issues founds
- Multimedia - no issues founds
- The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,199 +1,326 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
```
Comment (Bellagirl-maker) [inline index.html:7]: The page title was Document.....

File: index.html [hunk 2]
```diff
@@ -1,199 +1,326 @@
...
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Page</title>
```
Comment (Bellagirl-maker) [inline index.html]: We changed it to Portfolio Page

File: index.html [hunk 3]
```diff
@@ -1,199 +1,326 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
     <header class="navbar">
-        <a href="#" class="logo">My Logo</a>
-        <nav><img class="union" src="./images/Union.png"  alt="menu icon"></nav>
```
Comment (Bellagirl-maker) [inline index.html:16]: The img tag did not have a closing tag

File: index.html [hunk 4]
```diff
@@ -1,199 +1,326 @@
...
-        <ul class="menu-links">
-            <li><a href="#">Portfolio</a> </li>
-            <li><a href="#">About</a> </li>
-            <li><a href="#">Contact</a> </li>
-        </ul>
+      <a href="#" class="logo">My Logo</a>
+      <nav><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
```
Comment (Bellagirl-maker) [inline index.html:19]: We included a closing tag in the img tag.

File: index.html [hunk 5]
```diff
@@ -1,199 +1,326 @@
...
+      <ul class="menu-links">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
     </header>
     <main>
-        <section class="introduction">
-            <p class="first-intro">I’m Isabella<br>
```
Comment (Bellagirl-maker) [inline index.html:25]: This first-intro  class was a p tag.

File: index.html [hunk 6]
```diff
@@ -1,199 +1,326 @@
...
-                    <li><a href="https://github.com/Bellagirl-maker/"><img src="./images/github.png" alt="github icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/isabella-otoo-935901146/"><img src="./images/Linkedin icon.svg" alt="Linkedin icon"></a></li>
-                    <li><a href="#"><img src="./images/mail.svg" alt="email"></a></li>
-                    <li><a href="#"><img src="./images/Vector.svg" alt="Angellist"></a></li>
-                </ul>
-            </div> 
-        </section>
-    <section class="projects-section">
+      <section class="introduction">
+        <h1 class="first-intro">
```
Comment (Bellagirl-maker) [inline index.html:28]: We changed the first-intro class tag to h1 tag

File: index.html [hunk 7]
```diff
@@ -1,199 +1,326 @@
...
+            <li>
+              <a href="#"><img src="./images/Vector.svg" alt="Angellist" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="projects-section">
         <div class="card-works">
-            <img class="portfolio-card1" src="./images/snapshot1.png" alt="projects screenshot">
-            <img class="portfolio-desktop1" src="./images/Snapshoot Portfolio desktop1.png" alt="project screenshot">
```
Comment (Bellagirl-maker) [inline index.html:42]: The image class portfolio-card1 and portfolio-desktop1 had the same alt text "projects screenshoot

File: index.html [hunk 8]
```diff
@@ -1,199 +1,326 @@
...
-                <button class="normal-button">See Project</button>
+          <img
+            class="portfolio-card1"
+            src="./images/snapshot1.png"
+            alt="portfolio Mobile first image"
+          />
+          <img
+            class="portfolio-desktop1"
+            src="./images/Snapshoot Portfolio desktop1.png"
+            alt="portfolio Desktop first image"
```
Comment (Bellagirl-maker) [inline index.html:75]: We gave portfolio-card1 and portfolio-desktop1 different alt texts

File: index.html [hunk 9]
```diff
@@ -1,199 +1,326 @@
...
         </div>
         <div class="card-works">
-            <img class="portfolio-card1" src="./images/Snapshoot Portfolio 2.png" alt="project screenshot">
-            <div class="project-desc">
-                <p class="project-title">Multi-Post Stories</p>
-                <div class="frame2">
-                    <h3>CANOPY</h3>
-                    <img src="./images/Counter.png" alt="background image item">
-                    <h4>Back End Dev</h4>
-                    <img src="./images/Counter.png" alt="background image item">
```
Comment (Bellagirl-maker) [inline index.html:72]: img tags have the same alt text

File: index.html [hunk 10]
```diff
@@ -1,199 +1,326 @@
...
+            src="./images/Snapshoot Portfolio 2.png"
+            alt="portfolio mobile second image"
+          />
+          <div class="project-desc">
+            <p class="project-title">Multi-Post Stories</p>
+            <div class="frame2">
+              <h3>CANOPY</h3>
+              <img src="./images/Counter.png" alt="background image item3" />
+              <h4>Back End Dev</h4>
+              <img src="./images/Counter.png" alt="background image item4" />
```
Comment (Bellagirl-maker) [inline index.html:117]: We changed img tags to have different alt text

File: style.css [hunk 11]
```diff
@@ -92,7 +92,7 @@ a:hover {
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
-  color: #7f8cff;
```
Comment (Bellagirl-maker) [inline style.css:95]: The contrast ration didn't meet the requirement

File: style.css [hunk 12]
```diff
@@ -92,7 +92,7 @@ a:hover {
...
+  color: #303fc5;
```
Comment (Bellagirl-maker) [inline style.css:95]: We had to change the first color to this to pass the contrast ratio

File: index.html (hunk 2 above)
Comment (didierganthier) [reply index.html]: Very detailed report 👍🏽 but the `Portfolio` project requirement for the title is that It adequately and briefly describes the content of the page. For example:

### Poor titles:
```
Microverse project | Your portfolio.
```
or
```
Portifolio project
```

### Better page titles:
```
Mark's portfolio
```
or
```
Mark full-stack webdeveloper
```

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [x] Good job on making some parts of the website accessible 💯 but also make sure to document your README properly, some of the required sections were removed from the README, and the links they have in the Table of Contents are broken because of that. Kindly try to address that, please. 😉 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Good job on making some parts of the website accessible 💯 but also make sure to document your README properly, some of the required sections were removed from the README, and the links they have in the Table of Contents are broken because of that. Kindly try to address that, please. 😉 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Comment by Bellagirl-maker** [comment]
@didierganthier Can you please point out exactly which parts in the readme file I should include because I'm confused.

**Comment by didierganthier** [comment]
### Sure, take a look below, these are the required sections you should have in *Getting Started* 👇🏽 
Screenshot 2023-03-07 at 09 08 48

### Here you only have *Setup* so you're missing :
- [x] Prerequisites
- [x] Install
- [x] Usage
- [x] Run tests
- [x] Deployment

Screenshot 2023-03-07 at 09 08 24

File: index.html [hunk 13]
```diff
@@ -1,199 +1,326 @@
...
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Isabella Full-Stack Webdeveloper</title>
```
Comment (AFakhriddinov) [inline index.html:7]: Updated the title

**Review by BregornOriginal — APPROVED** [review]
## Hi @Bellagirl-maker 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

approve-applause

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**